astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -24,6 +25,18 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    # Execute AWS command then sleep for 5 min so that cluster would be available
+    create_redshift_cluster = BashOperator(
+        task_id="create_redshift_cluster",
+        bash_command="aws redshift create-cluster "
+        "--db-name dev "
+        "--cluster-identifier redshift-cluster-1 "
+        "--cluster-type single-node "
+        "--node-type dc2.large  "
```

Comment:
check if the node type can be a smaller one please?

Reply:
This is the cheapest one I'm able to find in the AWS dropdown.